src/proc/sequence_id_filter.cpp

```diff
@@ -30,61 +26,53 @@ namespace librealsense
         if (frame.is<rs2::frameset>())
             return false;
 
-        auto depth_frame = frame.as<rs2::depth_frame>();
-        if (!depth_frame)
-            return false;
-
-        if (!depth_frame.supports_frame_metadata(RS2_FRAME_METADATA_SUBPRESET_SEQUENCE_SIZE))
+        if (!frame.supports_frame_metadata(RS2_FRAME_METADATA_SEQUENCE_SIZE))
             return false;
-        if (!depth_frame.supports_frame_metadata(RS2_FRAME_METADATA_SUBPRESET_SEQUENCE_ID))
-            return false;
-        int depth_sequ_size = depth_frame.get_frame_metadata(RS2_FRAME_METADATA_SUBPRESET_SEQUENCE_SIZE);
-        if (depth_sequ_size == 0)
-            return false;
-
-        auto profile = frame.get_profile();
-        rs2_stream stream = profile.stream_type();
-        if (stream != RS2_STREAM_DEPTH)
+        if (!frame.supports_frame_metadata(RS2_FRAME_METADATA_SEQUENCE_ID))
             return false;
-
-        int index = profile.stream_index();
-        if (index != 0)
+        int sequ_size = frame.get_frame_metadata(RS2_FRAME_METADATA_SEQUENCE_SIZE);
```

Comment:
It is more common to use  `seq` and not `sequ` - pls rename for consistency

Reply:
Done